.env.example

```
@@ -1,9 +1,11 @@
 # Required
 NUXT_PUBLIC_SUPABASE_URL="https://example.supabase.co"
 NUXT_PUBLIC_SUPABASE_KEY="<your_publishable_key>"
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)